html/changelogs/Kaedwuff - Shoes.yml

```
@@ -0,0 +1,44 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: Kaedwuff
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "All soft and cloth footwear no longer clicks on the floor as you move. Certainy spooky or traitorous footwear is also affected by this. You may now stalk people in bunny slippers."
+  - tweak: "White shoes are no longer chemical resistant. Doctors and scientists are now equipped with special chem resistant shoes that are chem resistant, and distinctive from regular white shoes."
```

Should be worded better than "special chem resistant shoes, that are chem resistant"
I.e.
```suggestion
  - tweak: "White shoes are no longer chemical resistant. Doctors and scientists are now equipped with special chem resistant shoes which are distinctive from regular white shoes."
```